### DISCUSSION TITLE
Help needed understanding bid suggestions values from Sponsored Products keyword recommendations

### DISCUSSION THREAD

**Discussion by ricardodacosta** [amzn/ads-advanced-tools-docs#124] [Q&A]
The products I currently advertise have bid ranges between (0.50-4.50) according to when setting up campaigns via Amazon Seller Central. 

I have been working with Python to access and automate a lot of my tasks.

When using https://advertising-api.amazon.com/sp/targets/keywords/recommendations whether using :
"recommendationType": "KEYWORDS_FOR_ADGROUP" or
"recommendationType": "KEYWORDS_FOR_ASINS"

I am able to retrieve data but the bid ranges I am seeing are all in hundreds such as for example:
{'matchType': 'EXACT', 'rank': 1, 'bid': 128, 'suggestedBid': {'rangeStart': 96, 'rangeMedian': 128, 'rangeEnd': 160'}}

is it supposed to be 1.28, 0.96, 1.28, 1.60 ?

Am I doing something wrong or is the output of type integer and meant to be converted to float type ?

Thank you for your assistance!

Ricardo

**Comment by elizafarley** [comment]
Hi Ricardo,

These large bid ranges seems like a bug to me. I'd recommend opening a [support ticket](https://amzn-clicks.atlassian.net/servicedesk/customer/portal/2/group/2/create/6) so our engineering team can look into it.

**Reply by ricardodacosta** [reply]
Thank you. I submitted a ticket and will post the answer back here in case anyone else runs through similar issue
